auth/src/main/kotlin/com/hedvig/auth/services/UserService.kt

```diff
@@ -0,0 +1,265 @@
+package com.hedvig.auth.services
+
+import com.hedvig.auth.models.*
+import com.hedvig.auth.models.SwedishBankIdCredential
+import com.hedvig.auth.models.SwedishBankIdCredentialRepository
+import com.hedvig.auth.models.ZignSecCredential
+import com.hedvig.auth.models.ZignSecCredentialRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+/**
+ * The `UserService` can be used to lookup (and create or delete) a users. A user is associated
+ * with a `MemberId` and a unique set of credentials. The credentials can be used to lookup a user.
+ * To find a user with a matching credential you use the `findOrCreateUserWithCredential()`
+ * method.
+ *
+ * @see User
+ *
+ * @author Niklas Holmgren <niklas@hedvig.com>
+ * @author Fredrik Byström <fredrik.bystrom@hedvig.com>
```

Comment:
"Byström" is not my name 🙃 

```suggestion
 * @author Fredrik Tõnisson-Bystam <fredrik.bystam@hedvig.com>
```